Source/WarpX.H

```diff
@@ -150,6 +150,8 @@ public:
     static int  l_lower_order_in_v;
 
     static bool use_filter;
```

Comment:
can we please add documentation strings to those member variables?
e.g.
```suggestion
    static bool use_filter; //!< use a current filter (true) during evolve or not (false)
```

Reply:
This should be a separate PR. use_filter is not specific to the RZ spectral solver.